### DISCUSSION TITLE
all data storage in minio?

### DISCUSSION THREAD

**Discussion by WUWEI111222333** [databendlabs/databend#10754] [Q&A]
when deploy databend with minio,i create some tables,and insert some data

and if i deploy databend on other node with the same minio,the tables and data can not be found in databend

so my question is if one node down,i has to deploy on other node,how to keep my table and data exist

**Comment by WUWEI111222333** [comment]
i think i got it 
if i want to deploy on other node,i must backup and restore schema data,because the schema data is storage on local not  minio

**Reply by Xuanwo** [reply]
Databend stores its metadata inside Metasrv. Please make sure the data in metasrv had been dumped.

**Reply by WUWEI111222333** [reply]
Could you explain that a little bit,What is the Metasrv?

**Reply by sundy-li** [reply]
It's databend-meta
